content/2021/2021-01-21_updates-from-the-past-year/index.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Updates from the past year"
+description = "A lot has been done, a lot more to do!"
+
+[taxonomies]
+tags = ["announcement", "changelog", "runner", "new-language", "documentation", "moderation"]
++++
+
+
+For the past few years, we had been focused on the internal changes to fix critical bugs and performance issues. We've been aware of the various problems of the platform including the community issues, but it was necessary for us to prioritize those major technical issues. We're happy to announce that Codewars is now in a much better shape! Thank you for your patience.
+
+We started to focus more on the community aspect since the summer of 2020 and have some exciting announcements to make.
+In this post, we'll highlight the updates from the past year and explain our plan for the early 2021.
+
+
+## New blog
+
+Some of you might have noticed already, but we have a new blog. It's [open sourced](https://github.com/codewars/blog) on GitHub to collaborate with the community. Feel free to open issues for any feedback.
+
+We'll be posting about anything related to Codewars, including new features, improvements, and will ask for your help when needed. We're also considering guest posts by users to share their experience in the future.
+
+
+## New languages
+
+We've added support for the following languages: Prolog, CFML, COBOL, Lean, Haxe, CommonLisp, Perl, Raku, and Pascal.
+
+Most of them are still in the beta stage, and contributions in the form of new kata or translations will be very helpful. Many of the existing languages were also updated to newer versions. More details will be provided in a dedicated post.
+
+Many thanks to all users who provided their support on setting up the code runner and created new content for introduced languages!
+
+
+## Refactoring and performance improvements
+
+As mentioned above, we've been mostly focused on the technical issues for the past few years. We're relieved that _most_ of the critical issues have been resolved and the performance have improved significantly. In short, it's now at least 10 times faster on average compared to 2018 with less servers and much more traffic. Thank you for your patience and support if you had experienced the worst days and stayed with us.
+
+
+## Community moderation team
+
+The Codewars community needed moderation for a long time, but it was difficult because we only have one developer/admin. We wanted to delegate various tasks to a team of trusted users, but all the technical issues got in the way...
+
+We're excited to introduce [the community moderation team](https://docs.codewars.com/community/moderation/#members)! They have access to a variety of tools to manage content, handle problematic users, support users, and perform other moderation tasks. 
```

Comment:
```suggestion
We're excited to introduce [the community moderation team](https://docs.codewars.com/community/moderation/#moderators)! They have access to a variety of tools to manage content, handle problematic users, support users, and perform other moderation tasks. 
```
It was linked to the admin members. The moderation members have id `members-1`.

Reply:
I think there's still some problem with anchors, I am not sure though. Sometimes I am almost certain I navigate to some section, but the url in the address bar ends up being different.

I will see if I manage to confirm this reliably.